### ISSUE TITLE
Duplicating the API to accommodate regional spellings

### ISSUE THREAD

**Issue by nmsmith** [avh4/elm-color#18]
I just wanted to bring to discussion "grey" vs "gray" and attempting to accommodate regional spellings.

In my opinion, we should settle on one spelling and remove duplicate definitions from the API. Regional spellings don't really matter. As a pertinent example, much of the world spells color as "colour" (essentially all English speaking countries apart from the US), and that's the name of the core data type and the package itself! I think it would be more confusing for users to encounter multiple versions than a single version that may not use their preferred spelling.

I would suggest we give this one to the Brits and steal the spelling of Earl Grey. My two cents.

**Comment by avh4** [comment]
Yeah, I think I agree with this.  The reason the duplicates were included is because they existed in the Elm 0.17 elm/core Color module, so I brought them over.

I'll take this into consideration for the 2.0.0 release.

**Comment by johndeighan** [comment]
I think that "grey" is going to be a problem with folks from the US. The phrase "essentially all English speaking countries apart from the US" is a bit strange when, according to Wikipedia, there are about 400 million native speakers of English, of which 275 million are in the US. I don't see any reason that you can't have both "gray" and "grey" available. They're not going to conflict with other names.
